src/main/java/org/jabref/model/entry/Author.java

```diff
@@ -369,4 +356,16 @@ public String getNameForAlphabetization() {
         }
         return res.toString();
     }
+
+    /**
+     * Returns a LaTeX-free version of this `Author`.
+     */
+    public Author latexFree() {
+            String first = getFirst().map(LatexToUnicodeAdapter::format).orElse(null);
+            String firstabbr = getFirstAbbr().map(LatexToUnicodeAdapter::format).orElse(null);
+            String von = getVon().map(LatexToUnicodeAdapter::format).orElse(null);
+            String last = getLast().map(LatexToUnicodeAdapter::format).orElse(null);
+            String jr = getJr().map(LatexToUnicodeAdapter::format).orElse(null);
+            return new Author(first, firstabbr, von, last, jr);
```

Comment:
Does it make sense to cache the latex free author? Maybe it's better to do the caching here, instead in the authorlist class? That might be helpful, because then developers don't need to remember in which order to call things. For example, `authorList.getFirst(...).latexFree().getName()` works on the cache as well.

Reply:
I am torn on this issue. Earlier we discussed that list of `Author`s should, if possible, not be modified outside of `AuthorList`. It is too easy to make mistakes in parsing/reassembling authors and the code should be as DRY as possible (at least that was my takeaway, perhaps that is wrong).
What about moving `Author.latexFree` to `AuthorList` and making it private? Then there is only one "correct" option.

Reply:
Mhh... I think, it the responsibility of the author class to handle everything related to a single author; and the authorlist class for mulitple authors. Based on this, it would be the responsibility of the author class to handle caching of it's latexfree version. In my opinion, you also don't run into the kind of problems we discussed previously. Calling latexfree doesn't really modify the data in the class (all the important fields are final anyway). Do I miss something?

Reply:
> Do I miss something?

No, you are not. My argument boils down to that there is very little utility in using an individual `Author`, and perhaps it should be discouraged. I would view the method in `AuthorList` as a utility function rather than caching latex-free authors themselves.

Perhaps the difference between caching latex-free `Author`s and caching an `AuthorList` that happens to be latex-free is too small to bother about, I'll push a cached latex-free `Author`.

Reply:
> My argument boils down to that there is very little utility in using an individual Author, and perhaps it should be discouraged.

I agree very much with the first part of the statement, but wouldn't take this as a reason to discourage it's usage. Who knows what comes in the future. For now it's mainly a nice separation of concerns.

Thanks for your changes.

Reply:
Actually I already thought once of a special author editor dialog, but did not yet have time to dive into this deeper, since this is way down on the priority list. 😉

Reply:
😱 
Well, if you do get around to it, feel free to `@mention` me if I don't notice it on my own, I'd be interested in seeing the result 😛 
There are some low-priority pet peeves of mine that might be related to creating that functionality.

Reply:
Well, i thought of it like the Outlook name details dialog:
![grafik](https://user-images.githubusercontent.com/50491877/112686884-f30e0700-8e76-11eb-8396-3e31ce5ed983.png)

But this is probably just a nice little gimmick which is just pleasant to work on. If you like to work on this, go for it. :smile:

Reply:
It has been added to my todo-list but I don't think I'll be reaching it anytime soon :/